### PR TITLE
[x86/Linux] Enable FEATURE_ARRAYSTUB_AS_IL

### LINKED ISSUE #9687: [x86/Linux] PAL_SEHException is thrown while running b119538 tests
Here is the detailed logs from the test log:
```
FAILED   - [ 4997 :    3 sec ] JIT/Regression/CLR-x86-JIT/V1-M14-SP1/b119538/b119538a/b119538a.sh
               BEGIN EXECUTION
               /root/dotnet-test/coreclr_unittest_20170217_164016_x86_x86-emu/coreclr-unittest-170217-074046/overlay/corerun b119538a.exe
               terminate called after throwing an instance of 'PAL_SEHException'
               ./b119538a.sh: line 131:  2789 Aborted                 (core dumped) $_DebuggerFullPath "$CORE_ROOT/corerun" b119538a.exe $CLRTestExecutionArguments
               Expected: 100
               Actual: 134
               END EXECUTION - FAILED
FAILED   - [ 4998 :    3 sec ] JIT/Regression/CLR-x86-JIT/V1-M14-SP1/b119538/b119538b/b119538b.sh
               BEGIN EXECUTION
               /root/dotnet-test/coreclr_unittest_20170217_164016_x86_x86-emu/coreclr-unittest-170217-074046/overlay/corerun b119538b.exe
               terminate called after throwing an instance of 'PAL_SEHException'
               ./b119538b.sh: line 131:  2809 Aborted                 (core dumped) $_DebuggerFullPath "$CORE_ROOT/corerun" b119538b.exe $CLRTestExecutionArguments
               Expected: 100
               Actual: 134
               END EXECUTION - FAILED
```

### PR DESCRIPTION
**Pull request by parjong**
This commit enables FEATURE_ARRAYSTUB_AS_IL for x86/Linux to fix #9687.

### CODE REVIEW

**Comment by parjong** [comment]
This PR addresses the incorrect unwind issue discussed in #9688.

File: clr.coreclr.props [hunk 1]
```diff
@@ -83,6 +83,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetsUnix)' != 'true'">
+    <FeatureArrayStubAsIL Condition="('$(TargetArch)' == 'arm') or ('$(TargetArch)' == 'amd64') or ('$(TargetArch)' == 'arm64')">true</FeatureArrayStubAsIL>
```
Comment (jkotas) [inline clr.coreclr.props]: Nit: I would be nice to move `FeatureStubsAsIL` from the common place to here as well.

**Review by jkotas — APPROVED** [review]

File: clr.coreclr.props (hunk 1 above)
Comment (parjong) [inline clr.coreclr.props]: @jkotas Thanks you for comment! It would be much better. 7ec9019 revised PR as you suggested.

**Comment by parjong** [comment]
@dotnet-bot test Windows_NT x64 Release Priority 1 Build and Test please

**Comment by xiangzhai** [comment]
:mips-interest

**Pull request merged**